src/client/pages/Offer/PageWrapper.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { useHistory } from 'react-router'
+import { TopBar } from 'components/TopBar'
+import { Page } from 'components/utils/Page'
+import { useVariation, Variation } from 'utils/hooks/useVariation'
+import { trackOfferGTM, EventName } from 'utils/tracking/gtm'
+import { Features, useFeature } from 'utils/hooks/useFeature'
+import { useCurrentLocale } from 'l10n/useCurrentLocale'
+import { PhoneNumber } from 'components/PhoneNumber/PhoneNumber'
+import { OfferData } from 'pages/OfferNew/types'
+import { LanguagePicker } from '../Embark/LanguagePicker'
```

Comment:
nit: change to absolute import

Maybe we should move this outside Embark folder in the future